### DISCUSSION TITLE
Znp Robin nano 2.1 vs 2.2

### DISCUSSION THREAD

**Discussion by TheFeralEngineer** [NARUTOfzr/Neptune_3#61] [General]
Hello Naruto,

Can you ask your team what the difference is between pin 3 on j22 (leveling circuit) between v2.1 and v2.2? Specifically, pin 3 is linked to pa8 on the stm32f401 and I would like to know if I can change the firmware configuration on the neptune 3 pro and use pa8 for servo control to a BLTouch, like it is being used on the v2.1 board

![Screenshot_20230115-224642_Gallery](https://user-images.githubusercontent.com/74030381/212617319-0559ed57-fe35-4d45-8bd1-ee80927cd851.jpg)

**Comment by NARUTOfzr** [comment]
I think this is not feasible. This pin of Neptune3Pro motherboard v2.2 cannot be servo controlled.

**Reply by TheFeralEngineer** [reply]
Are there any pins that you think can be used for servo control? There are some people that want to use nonmetallic bed surfaces and cannot do this with an inductive sensor. There are also people crashing the nozzle into the magnet sheet because they aren't aware of this limitation.

**Comment by nerdtool** [comment]
I second this question.  I would like to be able to utilize glass or better yet Garolite G10 for my build plate but non-ferrous materials arent sensed by the inductive sensor.  Could I swap the inductive sensor with a capacitive one?  (TH3D makes several for other printers)

**Comment by TheFeralEngineer** [comment]
Yes, as long as it's 24v, it would be a direct swap. Only issue is fitting
it to the machine.

Phil T.
The Feral Engineer

www.theferalengineer.com

Catnip contributions 🐱🐱
https://paypal.me/timpsonmusicllc

**Comment by nerdtool** [comment]
Thanks for the speedy reply!
another question. looking at schematics for what I believe is the same ZNP
Robin Nano_DW v2.2 board, it looks like there are headers for connecting a
pin type (BL-Touch or CR-Touch) ABL sensor.  Is that also an option?  I
read somewhere that there are no servo headers to drive the raising and
lowering of the sensor pin in those ABL sensors.
Thanks again for your advice.
Ed (NerdTool) Waller

**Comment by TheFeralEngineer** [comment]
2.1 has servo controlled pins, 2.2 does not. I've managed to get servo
control from the dio pin on the service header for connecting an STLink but
I think that pin is being used for the x axis limit switch on the 2.2, but
I forget. It was either that board or the k1

Phil T.
The Feral Engineer

www.theferalengineer.com

Catnip contributions 🐱🐱
https://paypal.me/timpsonmusicllc
